graylog2-server/src/main/java/org/graylog2/inputs/codecs/JsonPathCodec.java

```diff
@@ -16,6 +16,9 @@
  */
 package org.graylog2.inputs.codecs;
 
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.*;
```

Comment:
We generally do not allow wildcard-imports, as they could introduce hard to find naming clashes. Instead, you should import the specific classes separately.